Header header
ObjPos[] targets_pos
